lib/module.js

```diff
@@ -90,6 +91,16 @@ function httpModule(_moduleOptions) {
       typeof options.proxy === 'object' ? options.proxy : {}
     ])
   }
+  // Add `ky` to build.transpile
+  this.options.build = this.options.build || {}
+  this.options.build.transpile = this.options.build.transpile || {}
+  // transpile only for non-modern build
+  // istanbul ignore if
+  if (semver.gte(this.nuxt.constructor.version, 'v2.9.0')) {
```

Comment:
This is not supporting edge versions like `2.9.0-26065925.ac00f7a6`.

![image](https://user-images.githubusercontent.com/5158436/61789215-5a192400-ae29-11e9-9f1c-0831f7c3a7ab.png)

Reply:
```suggestion
  if (semver.gte(semver.coerce(this.nuxt.constructor.version), '2.9.0')) {
```